Components/Alert/Alert.css

```diff
@@ -0,0 +1,65 @@
+.alert-container{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.alert-item{
+    padding: 2rem 4rem;
+}
+
+.success-alert{
+    background-color: var(--success-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--success-alert);
+    color: var(--success-font-color);
+}
+.danger-alert{
+    background-color: var(--danger-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--danger-alert);
+    color: var(--danger-font-color);
+}
+.info-alert{
+    background-color: var(--info-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--info-alert);
+    color: var(--info-font-color);
+}
+.warn-alert{
+    background-color: var(--warning-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--warning-alert);
+    color: var(--warning-font-alert);
+}
+
+.primary-alert{
+    background-color: var(--primary-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--primary-alert);
+    color: var(--primary-font-color);
+}
+.secondary-alert{
+    background-color: var(--secondary-alert);
+    margin: var(--alert-margin);
+    border-radius: var(--rem-border-radius);
+    border: 3px solid var(--secondary-alert);
+}
+
+.carbon-container{
+    max-width: 100%;
+    margin: 0 auto;
+}
```

Comment:
If this is common for all html pages , you can move to main css

Reply:
it will only common for components will create component.css for this and add it